tests/unit/helpers/balances.test.js

```diff
@@ -297,3 +300,150 @@ describe('getBalance', () => {
     expect(result.toPay).toEqual(0);
   });
 });
+
+describe('getBalances', () => {
+  let findBillsAmountsGroupedByClient;
+  let findCNAmountsGroupedByCustomer;
+  let findCNAmountsGroupedByTpp;
+  let findPaymentsAmountsGroupedByClient;
+  let getBalance;
+  let getBalancesFromCreditNotes;
+  let getBalancesFromPayments;
+
+  let customersAmounts;
+  let tppsAmounts;
+  let clientsAmounts;
+
+  const customers = [new ObjectID(), new ObjectID(), new ObjectID()];
+  const tpps = [new ObjectID(), new ObjectID()];
+
+  beforeEach(() => {
+    findBillsAmountsGroupedByClient = sinon.stub(BillRepository, 'findAmountsGroupedByClient');
+    findCNAmountsGroupedByCustomer = sinon.stub(CreditNoteRepository, 'findAmountsGroupedByCustomer');
+    findCNAmountsGroupedByTpp = sinon.stub(CreditNoteRepository, 'findAmountsGroupedByTpp');
+    findPaymentsAmountsGroupedByClient = sinon.stub(PaymentRepository, 'findAmountsGroupedByClient');
+    getBalance = sinon.stub(BalanceHelper, 'getBalance');
+    getBalancesFromCreditNotes = sinon.stub(BalanceHelper, 'getBalancesFromCreditNotes');
+    getBalancesFromPayments = sinon.stub(BalanceHelper, 'getBalancesFromPayments');
+
+    getBalance.returnsArg(0);
+    getBalancesFromCreditNotes.returnsArg(0);
+    getBalancesFromPayments.returnsArg(0);
+
+    customersAmounts = [
```

Comment:
pourquoi les mettre dans un `beforeEach` ? tu peux les définir avant directement en constante

Reply:
Je passe ces données a la fonction testée. C'est pour éviter le cas ou la fonction modifierait les données que je lui passe, on si je les modifies moi meme dans le `it`.
Bon ok dans ces tests ce n'est pas utile 😁.
Disons que c'est préventif, et que ca coute pas plus cher de l'ecrire comme ca je trouve.